src/library/tabledelegates/stardelegate.cpp

```diff
@@ -43,8 +59,15 @@ QWidget* StarDelegate::createEditor(QWidget* parent,
     QStyleOptionViewItem newOption = option;
     initStyleOption(&newOption, index);
 
-    StarEditor* editor =
-            new StarEditor(parent, m_pTableView, index, newOption, m_focusBorderColor);
+    StarEditor* editor = new StarEditor(parent,
```

Comment:
When using Qt fat pointer, we usually used our `parented_ptr` wrapper to help with visualising the ownership 

(Nit: rename `editor` and `parent` to `pEditor`, and `pParent` to help distinction to devs)

```suggestion
    auto pEditor = make_parented<StarEditor>(parent,
```

Reply:
Yeah, please use the pointer `p` prefix for all lines you touch, and `m_p` for pointer members.

Reply:
Will do. I just didn't have the time to update the code yet 😅